performant_PoCs/focused-vision/focused_pipeline/main.py

```diff
@@ -0,0 +1,156 @@
+from pathlib import Path
+
+import depthai as dai
+from depthai_nodes.node import GatherData, ParsingNeuralNetwork, ImgDetectionsBridge
+from depthai_nodes.node.utils import generate_script_content
+
+from utils.annotation_node import AnnotationNode
+from utils.grid_eyes_annotation_node import GridEyesAnnotationNode
+from utils.arguments import initialize_argparser
+from utils.grid_layout_node import GridLayoutNode
+
+_, args = initialize_argparser()
+
+INPUT_WIDTH, INPUT_HEIGHT = 3840, 2160
+DET_MODEL = "luxonis/yunet:320x240"
+EYE_MODEL = "luxonis-ml-team/eye-detection:eye-detection-512x512"
+
+visualizer = dai.RemoteConnection(httpPort=8082)
+device = dai.Device(dai.DeviceInfo(args.device)) if args.device else dai.Device()
+platform = device.getPlatform().name
+
+frame_type = dai.ImgFrame.Type.BGR888i if platform == "RVC4" else dai.ImgFrame.Type.BGR888p
+
+if not args.fps_limit:
+    args.fps_limit = 8 if platform == "RVC2" else 30
+
+with dai.Pipeline(device) as pipeline:
+    det_desc = dai.NNModelDescription(DET_MODEL)
```

Comment:
use more descriptive variable name. Avoid using shortcuts in variable names. In this case, `head_detection_model_description` would be nice